BackendApartmentReservation/BackendApartmentReservation/Controllers/ProfilesController.cs

```diff
@@ -8,11 +16,38 @@ namespace BackendApartmentReservation.Controllers
     [ApiController]
     public class ProfilesController : ControllerBase
     {
+        private readonly UserManager<DbEmployee> _userManager;
         [HttpGet]
         [Route("profiles")]
         public async Task<IEnumerable<string>> Get()
         {
             return await Task.FromResult(new[] { "userProfile1", "userProfile2" });
         }
+
+        [HttpPost]
+        [AllowAnonymous]
+        [Route("register")]
+        public async Task<IActionResult> Register([FromBody] RegisterRequest model)
+        {
+            if (!ModelState.IsValid) return BadRequest(ModelState);
+            if (RegistrationInputValidator.IsInputValid(model)) return BadRequest("Input is not valid");
```

Comment:
No human readable error codes should be returned to the frontend in my opinion. Return an error string (error code) which contains no spaces, `!`s or any other special non-latin chars except `_`. Let the frontend handle localisation of that error code.

For the reference, take a look at my global exception handler.